docs/security/auth/maskinporten.md

```diff
@@ -162,19 +164,125 @@ The following describes the steps needed to migrate a client registered in [IaC
 - Verify that everything works after the migration
 - Delete the application from the [IaC repository](https://github.com/navikt/nav-maskinporten) in order to maintain a single source of truth.
 
-## Internals
+## Expose Scopes
+
+### About
 
-See [ID-porten internals](idporten.md#internals).
+In Maskinporten context, an API is the same as an Oauth2 scope. NAV as API provider has a prefix manually assigned. 
```

Comment:
Nevnte det i sted, men det kan være lurt å gi leseren litt mer kontekst. F.eks. nevne bruksområder for eksponering av scopes. Hvorfor ønsker man å eksponere et scope? Hva betyr det å eksponere et scope? Hvordan påvirker det oss som tlibyder og evt. hvordan påvirker det konsumenter av et scope?

Reply:
enig i de